subsys/mgmt/updatehub/include/updatehub.h

```diff
@@ -69,6 +69,23 @@ enum updatehub_response updatehub_probe(void);
  */
 enum updatehub_response updatehub_update(void);
 
+/**
+ * @brief Confirm that image is running as expected.
+ *
+ * @details Must be used before the UpdateHub probe. It should be one of first
+ * actions after reboot.
+ *
+ * @return Return 0 if success otherwise a negative @errorno value.
+ */
+int updatehub_confirm(void);
```

Comment:
This text would be useful as part of the generated documentation, not blocking it in this PR but ideally this (and other public functions) should move to `zephyr/include/zephyr/mgmt/*` so that it gets generated, with a clean include file path too.

Reply:
I'll open another PR to do this suggestion.

Thank you for the hint @nordicjm!

Reply:
Addressed at https://github.com/zephyrproject-rtos/zephyr/pull/53635